container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -174,10 +183,18 @@ instance_groups:
         - name: sched-health
           protocol: TCP
           internal: 8080
+  - name: bpm
+    release: bpm
+    properties:
+      bosh_containerization:
+        run:
+          capabilities: [ALL]
+        pod-security-policy: privileged
```

Comment:
For new manifests we should say `service-account` instead of `pod-security-policy`. (Because trying to do PSPs here directly is incorrect; PSPs are tied to service accounts)

Reply:
Ok. Can do that.

Reply:
Done for all uses, with commit c5c618134e4d775c67f8d2b8b23fe9f450ca2873.
Separate commit, to make it easy to undo in case of trouble.